securedrop_client/gui/main.py

```diff
@@ -100,47 +101,47 @@ def show_main_window(self, username: str = None) -> None:
         if username:
             self.set_logged_in_as(username)
 
-    def autosize_window(self):
+    def autosize_window(self) -> None:
         """
         Ensure the application window takes up 100% of the available screen
         (i.e. the whole of the virtualised desktop in Qubes dom)
         """
         screen = QDesktopWidget().screenGeometry()
         self.resize(screen.width(), screen.height())
 
-    def show_login(self):
+    def show_login(self) -> None:
         """
         Show the login form.
         """
-        self.login_dialog = LoginDialog(self)
+        self.login_dialog = LoginDialog(self) # type: Optional[LoginDialog]
```

Comment:
Small refactor: We should define the type of this in the `__init__` method so that on visual inspection of the class, all defined attributes are in one place. Additionally, setting it to `None` during `__init__` prevents the case where accessing it would raise an `AttributeError` if we don't call `setup`, such as in testing. I know this slightly extends the scope of your work (minor refactor, not just adding annotations).